contract/src/main/java/io/neow3j/contract/ContractManagement.java

```diff
@@ -61,6 +64,19 @@ public TransactionBuilder setMinimumDeploymentFee(BigInteger minimumFee) {
         return invokeFunction(SET_MINIMUM_DEPLOYMENT_FEE, integer(minimumFee));
     }
 
+    /**
+     * Checks if a method exists in a contract.
+     *
+     * @param contractHash the contract hash.
+     * @param method       the method.
+     * @param paramCount   the number of parameters.
+     * @return true if the method exists. False otherwise.
+     * @throws IOException if there was a problem fetching information from the Neo node.
+     */
+    public boolean hasMethod(Hash160 contractHash, String method, int paramCount) throws IOException {
```

Comment:
Should also be covered by a test in `io.neow3j.contract.ContractManagementIntegrationTest`.

Reply:
Added